src/main/java/io/stargate/sgv2/jsonapi/service/shredding/tables/RowShredder.java

```diff
@@ -1,20 +1,27 @@
 package io.stargate.sgv2.jsonapi.service.shredding.tables;
 
-import com.datastax.oss.driver.api.core.CqlIdentifier;
-import com.datastax.oss.driver.api.core.metadata.schema.ColumnMetadata;
 import com.fasterxml.jackson.databind.JsonNode;
-import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import io.stargate.sgv2.jsonapi.api.model.command.clause.filter.JsonLiteral;
+import io.stargate.sgv2.jsonapi.api.model.command.clause.filter.JsonType;
 import io.stargate.sgv2.jsonapi.api.v1.metrics.JsonProcessingMetricsReporter;
 import io.stargate.sgv2.jsonapi.config.DocumentLimitsConfig;
-import io.stargate.sgv2.jsonapi.service.cqldriver.executor.TableSchemaObject;
-import io.stargate.sgv2.jsonapi.service.resolver.UnvalidatedClauseException;
+import io.stargate.sgv2.jsonapi.service.shredding.JsonNamedValue;
+import io.stargate.sgv2.jsonapi.service.shredding.JsonNamedValueContainer;
+import io.stargate.sgv2.jsonapi.service.shredding.OrderedJsonNamedValueContainer;
+import io.stargate.sgv2.jsonapi.service.shredding.collections.JsonPath;
 import jakarta.enterprise.context.ApplicationScoped;
 import jakarta.inject.Inject;
+import java.util.ArrayList;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
 
 /**
- * AARON TODO shreds docs for rows
+ * Shreds a document transforming it from a {@link JsonNode} to a {@link
```

Comment:
Not sure if it's too late, but "Row" here is slightly confusing as it implies this would be for Tables, but it is actually for Collections based on comments, and refs to `JsonNamedValueContainer`.
But then again, class is in "...shredding.tables" package.

I am confused. :)

Reply:
Reading back and forth: this is just for Tables, but there's initial stage of "JSON-oriented" representation used, before we map to schema. That's ok (although there's obv some overhead for intermediate throw-away representation) but I think we can simplify: `JsonLiteral` adds no value over "raw" java.lang.Objects (just wrapper).
`JsonPath` is also overkill as we really just want flat names; keeping as `String` seems preferable -- names within `Map`s should not have full paths anyway.

Reply:
Based on discussions makes more sense: `JsonLiteral` is thin wrapper that we'll keep; `JsonPath` fine for now but can re-evaluate based on work wrt Filters and (update) Operations.